### ISSUE TITLE
Unable to connect to mysql database with mysql-connector-python with created users -- from stack overflow

### ISSUE THREAD

**Issue by Chaoui-lpb** [lavsharmaa/lavsharmaa#1]
Then I still have problem with the MySQL syntax with the line code that you gave to me
```sql
UPDATE user SET authentication_string=password('password') WHERE user='user2';
```

```You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '('password') WHERE user='user2'' at line 1```
I don't really know the Mysql syntax then I don't really know what you want to do

**Comment by lavsharmaa** [comment]
I am just trying to update the password nothing else, if there is some issue with your password or something like that.
```CREATE USER 'newuser'@'localhost' IDENTIFIED BY 'password';```
```GRANT ALL PRIVILEGES ON * . * TO 'newuser'@'localhost';```
```FLUSH PRIVILEGES;```
after doing this close the application as whole and then try connecting using ```newuser```.
Try to print(cnx) after the mysql.connector.connect line

**Comment by lavsharmaa** [comment]
Try to print(cnx) after the ```mysql.connector.connect``` line

**Comment by Chaoui-lpb** [comment]
It still doesn't work. I even tried to put it directly in the SQL file. 
Moreover, I can't print(cnx) because I'm in try-except clause. If I try to delete the try-except clause I can't print(cnx) because I have this error : ```mysql.connector.errors.ProgrammingError: 1045 (28000): Access denied for user 'user2'@'172.19.0.1' (using password: YES)```

**Comment by lavsharmaa** [comment]
I don't think so print statement got anything to do with your try-except clause. It is not going to give you any error while printing. And talking about access denied it is just because of lack of permission nothing else.

**Comment by Chaoui-lpb** [comment]
But it doesn't make any sense if I give all permissions to user2. He should be able to connect to database.

**Comment by lavsharmaa** [comment]
It should but for some other reason or due to some privilege issue it is not happening.
